wpeverest/user-registration
# Addon - User Registration Pro

### All Submissions:






### Changes proposed in this Pull Request:
Lately, we have developed `User Registration PRO` addon, and for premium users, this add-on acts as a combination of Free version and PRO version features so Premium users must activate this addon in order to smoothly use our addons and deactivate and can also delete User Registration Plugin's Free version. So this PR contains the codes of compatibility for the PRO version.

### How to test the changes in this Pull Request:

1. Activate PRO addon and check all the possible scenarios that could happen after activating PRO i.e notices before activating PRO, License Activation, and all the Features of Free versions and Addons.

### Types of changes:




### Other information:




### Changelog entry

> Addon - User Registration Pro.
